Move `zoom-frm` package locally since it disappeared from elpa

Robdel12 opened the pull request:
## What is this?

This PR should fix #115. The issue is the `zoom-frm` has disappeared from elpa & melpa. I'm not quite sure how I feel about copy & pasting the package source into our project.

~I'd like to figure out why this package has disappeared from the package sources.~

Turns out it was a wiki package & melpa removed wiki sourced packages https://github.com/melpa/melpa/pull/5008

Robdel12 committed:
Move `zoom-frm` package locally since it disappeared from elpa

cowboyd commented:
can we put it in `vendor/zoom-frm.el` with a note about why this was extracted?

Robdel12 commented:
@cowbyd absolutely. 

How hard would it be start building this on circle CI?

cowboyd commented:
Honestly since Circle 2.0 supports BYOC (bring your own container) It should be a lot easier. We can just have a container for each version of emacs we want to build against.

Robdel12 commented:
Oh my, we should absolutely do that

Robdel12 committed:
Move `zoom-frm` package to vendor folder

Robdel12 commented:
@cowboyd I moved the file to `vendor`.

cowboyd commented on frontmacs-keys.el:
```diff
@@ -17,6 +17,7 @@
 (require 'comment-dwim-2)
 (require 'browse-kill-ring)
 (require 'counsel)
+(require 'zoom-frm)
```
How does `vendor` get added to the load-path? Wouldn't this need to be changed in `frontside.el` or `frontside-config.el`?

Robdel12 replied:
That was going to be my next question. I'll update

cowboyd replied:
Does it work when you run init-test?

cowboyd committed:
Vendor zoom-frm dependencies.

zoom-frm depends on both frame-fns and frame-cmds. This incorporates
both of those packages and makes Frontmacs reference those instead.

cowboyd committed:
Try a travis hack mebbe?
